source/docs/quickstart.md

```diff
@@ -86,13 +85,29 @@ Here's how to get started with Atomic on your machine using virt-manager on Linu
 
 **Note**: When running virt-manager on Red Hat Enterprise Linux 6 or CentOS 6, the VM **will not boot** until the disk storage format is changed from raw to qcow2.
 
-
 #### Adding the CD-ROM device for the metadata source
 
 1. In the virt-manager GUI, click to open your Atomic machine. Then on the top bar click *View > Details*
 2. Click on *Add Hardware* on the bottom left corner.
 3. Choose *Storage*, and *Select managed or other existing storage*. Browse and select the `init.iso` image you created. Change the *Device type* to CD-ROM device.  Click on *Finish* to create and attach this storage.
 
+### Creating with virt-install (Command-Line Method)
+1. Use the following example virt-install command, **modifying each parameter to fit
+your requirements**.  In this example, a new directory, `Fedora-Atomic-26/`, was created under `/var/lib/libvirt/images` so that the `init.iso` created earlier, the vm image `Fedora-Atomic-26-20170723.0.x86_64.qcow2`, the `meta-data` file, and the `user-data` file are all in one folder.
+```
+virt-install --name atomic-host \
+--description 'Fedora Atomic Host' \
+--ram 4096 \
+--vcpus 4 \
+--disk path=/var/lib/libvirt/images/Fedora-Atomic-26/Fedora-Atomic-26-20170723.0.x86_64.qcow2 \
+--os-type linux \
+--os-variant fedora25 \ # at the time of this writing, fedora26 has not yet been implemented as an option
```

Comment:
Could you put your comments following the hash tags above or below this command to allow the user to cut and paste? These comments broke cut and paste for me.